ether_addresses.txt

```diff
@@ -0,0 +1,7 @@
+github id: 843
+ether address: 0xD4a64570B12dA659Ee4BBd41c3509B7b1F9c51AC (Metamask)
+
+               0x0020e20e6C9ACc0c9fE78Fcb6a427Ad0cC8aca18 (Parity)
+
+               0xD4a64570B12dA659Ee4BBd41c3509B7b1F9c51AC (kosmos parity)
```

Comment:
Is this a personal notes file that was accidentally committed?

Reply:
Damn, yes. This wasn't supposed to be committed.